docs/source/1.0/spec/core/constraint-traits.rst

```diff
@@ -606,13 +606,15 @@ in a response.
 
 .. warning:
 
-    This trait has been deprecated and will be removed in future versions of
-    Smithy. Set shapes should be used instead.
+    This trait has been deprecated. It may be removed in future versions. Set
+    shapes should be used instead.
 
 Summary
     Indicates that the items in a :ref:`list` MUST be unique.
 Trait selector
-    ``:test(list > member > :test(string, blob, byte, short, integer, long, bigDecimal, bigInteger))``
+    ``:test(list > member > simpleType)``
```

Comment:
why is this different than in the prelude?

Reply:
I straight up did a revert commit without even looking at the contents. I don't know.

Reply:
ah, so was incorrect before too. I'm fine with the revert commit, but maybe another commit to fix the documentation.